.vscode/launch.json

```diff
@@ -5,8 +5,8 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "OpenAI Example",
-            "cwd": "examples/openai",
```

Comment:
Can you please keep both entries in this configuration array? We still need it for the OpenAI example app.

Reply:
right, my apologies